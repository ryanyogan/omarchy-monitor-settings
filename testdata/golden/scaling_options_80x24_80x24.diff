Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [32m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;32m🧠 Smart Scaling Recommendations[0m                                      [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [33m╭─────────────────────────────────────────╮[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz[0m[40m  [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m╰─────────────────────────────────────────╯[0m                           [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;36m🎯 Available Options[0m                                                  [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;34m▶ [0m[1;34mStandard (1x) [RECOMMENDED][0m                                         [32m│[0m    
  [32m│[0m  [37m    Default scaling for sharp text[0m                                    [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080[0m        [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                               [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m    Large (1.25x)                                                       [32m│[0m    
  [32m│[0m  [37m    Comfortable scaling for most users[0m                                [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864[0m        [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                               [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;35m📚 What Each Setting Does[0m                                             [32m│[0m    
  [32m│[0m  [37m  Monitor Scale: Changes compositor-level scaling (immediate effect)[0m  [32m│[0m    
  [32m│[0m  [37m  GTK Scale: Scales GTK applications (requires logout/login)[0m          [32m│[0m    
  [32m│[0m  [37m  Font DPI: Fine-grained text scaling (affects most apps)[0m             [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [32m↑↓[0m[37m select[0m  [34m⏎[0m[37m apply[0m  [33mm[0m[37m manual[0m  [35mesc[0m[37m back[0m                                [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                                                                        │    
  │  🧠 Smart Scaling Recommendations                                      │    
  │                                                                        │    
  │  ╭─────────────────────────────────────────╮                           │    
  │  │                                         │                           │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz  │                           │    
  │  │                                         │                           │    
  │  ╰─────────────────────────────────────────╯                           │    
  │                                                                        │    
  │  🎯 Available Options                                                  │    
  │                                                                        │    
  │  ▶ Standard (1x) [RECOMMENDED]                                         │    
  │      Default scaling for sharp text                                    │    
  │      Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080        │    
  │      💡                                                                │    
  │                                                                        │    
  │    Large (1.25x)                                                       │    
  │      Comfortable scaling for most users                                │    
  │      Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864        │    
  │      💡                                                                │    
  │                                                                        │    
  │                                                                        │    
  │  📚 What Each Setting Does                                             │    
  │    Monitor Scale: Changes compositor-level scaling (immediate effect)  │    
  │    GTK Scale: Scales GTK applications (requires logout/login)          │    
  │    Font DPI: Fine-grained text scaling (affects most apps)             │    
  │                                                                        │    
  │  ↑↓ select  ⏎ apply  m manual  esc back                                │    
  │                                                                        │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 6060
Actual length: 4900
Expected hash: c2304f2a8951cad9c76ac317df7a885319f7ab54a748e27d0c63054dadf88776
Actual hash: 2758b92ef09491be8828234b117d345ebf3c03166ad32fc24e3230b0bb02120d
