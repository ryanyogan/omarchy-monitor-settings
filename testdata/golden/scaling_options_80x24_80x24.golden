# Visual Golden File
# Name: scaling_options_80x24
# Dimensions: 80x24
# Hash: c2304f2a8951cad9c76ac317df7a885319f7ab54a748e27d0c63054dadf88776

  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [32m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;32m🧠 Smart Scaling Recommendations[0m                                      [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [33m╭─────────────────────────────────────────╮[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz[0m[40m  [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                           [32m│[0m    
  [32m│[0m  [33m╰─────────────────────────────────────────╯[0m                           [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;36m🎯 Available Options[0m                                                  [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;34m▶ [0m[1;34mStandard (1x) [RECOMMENDED][0m                                         [32m│[0m    
  [32m│[0m  [37m    Default scaling for sharp text[0m                                    [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080[0m        [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                               [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m    Large (1.25x)                                                       [32m│[0m    
  [32m│[0m  [37m    Comfortable scaling for most users[0m                                [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864[0m        [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                               [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [1;35m📚 What Each Setting Does[0m                                             [32m│[0m    
  [32m│[0m  [37m  Monitor Scale: Changes compositor-level scaling (immediate effect)[0m  [32m│[0m    
  [32m│[0m  [37m  GTK Scale: Scales GTK applications (requires logout/login)[0m          [32m│[0m    
  [32m│[0m  [37m  Font DPI: Fine-grained text scaling (affects most apps)[0m             [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m  [32m↑↓[0m[37m select[0m  [34m⏎[0m[37m apply[0m  [33mm[0m[37m manual[0m  [35mesc[0m[37m back[0m                                [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m│[0m                                                                        [32m│[0m    
  [32m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    